<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Tourist_add_form</class>
 <widget class="QWidget" name="Tourist_add_form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>249</width>
    <height>134</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Добавление туриста</string>
  </property>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>10</y>
     <width>241</width>
     <height>116</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="tourist_ID">
      <property name="text">
       <string>ID туриста</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="tourist_name">
      <property name="text">
       <string>ФИО</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="tourist_birthday">
      <property name="text">
       <string>Дата рождения</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0" colspan="2">
     <widget class="QPushButton" name="addition">
      <property name="text">
       <string>Добавить</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="tourist_ID_input"/>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="tourist_name_input"/>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="tourist_birthday_input"/>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
